src/Kiota.Builder/Extensions/StringExtensions.cs

```diff
@@ -53,7 +53,7 @@ public static string ToPascalCase(this string? input, params char[] separators)
     /// </summary>
     /// <param name="fileName">The file name to shorten</param>
     /// <param name="maxFileNameLength">The maximum length of the file name. Default 251 = 255 - .ext</param>
-    public static string ShortenFileName(this string name, int length = 251) =>
+    public static string ShortenFileName(this string name, int length = 100) =>
```

Comment:
This is also going to impact other languages. 
https://github.com/microsoft/kiota/blob/main/src/Kiota.Builder/PathSegmenters/CSharpPathSegmenter.cs#L11
https://github.com/microsoft/kiota/blob/main/src/Kiota.Builder/PathSegmenters/CSharpPathSegmenter.cs#L12

Can I suggest passing a value from where it's called in Go instead please?
https://github.com/microsoft/kiota/blob/ca0a96b5fb189a7032216050f18c40ddb2119c0a/src/Kiota.Builder/PathSegmenters/GoPathSegmenter.cs#L88